From 0bc1b2e3f8d519a01505e428881a348b5da9bbb8 Mon Sep 17 00:00:00 2001
From: bernard <bernard.xiong@rt-thread.com>
Date: Sat, 1 Jul 2017 11:57:36 +0800
Subject: [PATCH] cpu_start patch

---
 components/esp32/cpu_start.c | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/components/esp32/cpu_start.c b/components/esp32/cpu_start.c
index 858036c6..711cbe76 100644
--- a/components/esp32/cpu_start.c
+++ b/components/esp32/cpu_start.c
@@ -242,16 +242,17 @@ void start_cpu0_default(void)
     esp_setup_time_syscalls();
     esp_vfs_dev_uart_register();
     esp_reent_init(_GLOBAL_REENT);
-#ifndef CONFIG_CONSOLE_UART_NONE
-    const char* default_uart_dev = "/dev/uart/" STRINGIFY(CONFIG_CONSOLE_UART_NUM);
-    _GLOBAL_REENT->_stdin  = fopen(default_uart_dev, "r");
-    _GLOBAL_REENT->_stdout = fopen(default_uart_dev, "w");
-    _GLOBAL_REENT->_stderr = fopen(default_uart_dev, "w");
-#else
+
     _GLOBAL_REENT->_stdin  = (FILE*) &__sf_fake_stdin;
     _GLOBAL_REENT->_stdout = (FILE*) &__sf_fake_stdout;
     _GLOBAL_REENT->_stderr = (FILE*) &__sf_fake_stderr;
-#endif
+
+    extern void rtthread_startup(void);
+    extern int rtthread_stdio_init(void);
+
+    rtthread_startup();
+    rtthread_stdio_init();
+
 #if CONFIG_ESP32_APPTRACE_ENABLE
     esp_err_t err = esp_apptrace_init();
     if (err != ESP_OK) {
-- 
2.11.0.windows.3

