From 4532e8a399323bbf2c68759d21e8de4dad3c4f5a Mon Sep 17 00:00:00 2001
From: Bernard Xiong <bernard.xiong@gmail.com>
Date: Thu, 20 Jul 2017 07:37:00 +0800
Subject: [PATCH] add esp_task.h

---
 components/esp32/cpu_start.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/esp32/cpu_start.c b/components/esp32/cpu_start.c
index 711cbe76..0fc0ef75 100644
--- a/components/esp32/cpu_start.c
+++ b/components/esp32/cpu_start.c
@@ -43,6 +43,7 @@
 #include "esp_heap_alloc_caps.h"
 #include "sdkconfig.h"
 #include "esp_system.h"
+#include "esp_task.h"
 #include "esp_spi_flash.h"
 #include "nvs_flash.h"
 #include "esp_event.h"
-- 
2.11.0.windows.3

